### PR TITLE
Profiles: ENS Registration Action Buttons

### PR DESCRIPTION
**Pull request by jxom**
Fixes RNBW-2290

## What changed (plus any additional context for devs)

This PR adds the "Continue on" & "Clear" sheet action buttons into the ENS registration flow

## PoW (screenshots / screen recordings)

https://www.loom.com/share/48274af434324a43901696b868f6fb7e

## Dev checklist for QA: what to test

- "Continue on" appears when an ENS is available
- "Clear" appears when ENS is not available
- Pressing "Clear" clears the input

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels?

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -1065,6 +1065,6 @@ SPEC CHECKSUMS:
   ToolTipMenu: 8ac61aded0fbc4acfe7e84a7d0c9479d15a9a382
   Yoga: 2b4a01651f42a32f82e6cef3830a3ba48088237f
 
-PODFILE CHECKSUM: 57c94b75db7f46e64a9d6be6e7513d7bcdaf6cb8
+PODFILE CHECKSUM: d711ded560059f2a44650df3dc65f13c99bdb226
```
Comment (jxom) [inline ios/Podfile.lock:1068]: ignore this, ill discard

File: ios/Rainbow.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -722,7 +722,7 @@
 			);
 			inputPaths = (
 				"${PODS_ROOT}/Target Support Files/Pods-Rainbow/Pods-Rainbow-frameworks.sh",
-				"${PODS_XCFRAMEWORKS_BUILD_DIR}/hermes/hermes.framework/hermes",
+				"${PODS_XCFRAMEWORKS_BUILD_DIR}/hermes-engine/hermes.framework/hermes",
```
Comment (jxom) [inline ios/Rainbow.xcodeproj/project.pbxproj:725]: ignore this, ill discard

File: src/components/buttons/TintButton.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+  const height = 56;
+
+  return (
+    <Box
+      alignItems="center"
+      as={ButtonPressAnimation}
+      height={`${height}px`}
+      justifyContent="center"
+      // @ts-expect-error
```
Comment (estebanmino) [inline src/components/buttons/TintButton.tsx:28]: just out of curiosity, what's this comment doing?

**Review by benisgold — APPROVED** [review]

**Review by estebanmino — APPROVED** [review]
looking good in both OSs

**Pull request merged**